addon/modifiers/sortable-item.js

```diff
@@ -296,6 +302,9 @@ export default class SortableItemModifier extends Modifier {
    * @private
    */
   _primeDrag(startEvent) {
+    // Prevent dragging if the entire group is disabled
+    if (this.disabled) { return; }
+
     // Prevent dragging if the sortable-item is disabled.
     if (this.isDraggingDisabled) {
```

Comment:
I agree with you, `isDraggingDisabled` should also be added for keyboard.
If you can create an issue as a follow up for `isDraggingDisabled` and put up a dedicated PR, that'd be nice. I just don't want the followup to be forgotten.
Although another concern is if we rename `isDragginDisabled` that might require a major release since it is a breaking change, so I'm debating the following options:
1. keeping the name for now, even tho misleading, to avoid major release. We can rename it along with future planned major release.
2. Change name and mark as major release.
3. Change name and mark as minor release and some additional warnings around the name migration

Reply:
I think we can support `isDraggingDisabled` and `disabled` in parallel for now to give people a chance to switch over. Then whenever we do a 3.0 release we can remove it.

Reply:
> give people a chance to switch over. Then whenever we do a 3.0 release we can remove it

Just to make sure we're on the same page, we'll support both `isDraggingDisabled` for individual items and `disabled` for groups for now. In the future, we are just renaming `isDraggingDisabled` to something more reasonable.

The inability to disable via keyboard is a bug and will be addressed as a separate PR?

Reply:
I will add `disabled` to the items as well and support both `isDraggingDisabled` and `disabled` on items. I will fix `isDraggingDisabled` to also prevent keyboard resorting, but in 3.0 it will be removed in favour of just `disabled`.